### PR TITLE
Adds 1 Synths job slot every 40 pop

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
This is a revive of https://github.com/tgstation/TerraGov-Marine-Corps/pull/6333

## Why It's Good For The Game
"As we are constantly moving towards high pop, I believe synthetics getting more slot every 40 pop makes sense as generally there is a lack of command. Synthetics can do most shipside job, and as the pop increases, it generally gets difficult to manage shipside. Addition of a support role of Synthetic will ensure this.

I may change the pop number upon testing and feedback. This needs testing."

## Changelog
:cl:
balance: Gives 1 Synth job slot every 40 pop.
/:cl:

### CODE REVIEW

File: code/datums/jobs/job/shipside.dm [hunk 1]
```diff
@@ -738,11 +750,11 @@ Use your office fax machine to communicate with corporate headquarters or to acq
 	exp_requirements = XP_REQ_EXPERIENCED
 	exp_type = EXP_TYPE_REGULAR_ALL
 	job_flags = JOB_FLAG_SPECIALNAME|JOB_FLAG_LATEJOINABLE|JOB_FLAG_ROUNDSTARTJOINABLE|JOB_FLAG_ADDTOMANIFEST|JOB_FLAG_ISCOMMAND|JOB_FLAG_BOLD_NAME_ON_SELECTION|JOB_FLAG_PROVIDES_SQUAD_HUD
+	job_points_needed = 40
 	jobworth = list(
 		/datum/job/xenomorph = LARVA_POINTS_SHIPSIDE_STRONG,
 		/datum/job/terragov/squad/specialist = SPEC_POINTS_REGULAR,
-		/datum/job/terragov/squad/smartgunner = SMARTIE_POINTS_REGULAR,
-	)
+		/datum/job/terragov/squad/smartgunner = SMARTIE_POINTS_REGULAR,)
```
Comment (BraveMole) [inline code/datums/jobs/job/shipside.dm:757]:
```suggestion
		/datum/job/terragov/squad/smartgunner = SMARTIE_POINTS_REGULAR,
	)
```

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by SpaceLoveSs13** [comment]
Thanks for getting my PR revived. Like the reason is mentioned above, this pr is good.

**Comment by fernandos13** [comment]
FINALLY THANKS!!!

**Comment by Losenis** [comment]
MERGE WHEN

**Comment by Pariah919** [comment]
DELETE SYNTH

**Comment by TiviPlus** [comment]
Go home MJP
Is this tested?

**Comment by SpaceLoveSs13** [comment]
> Go home MJP
> Is this tested?

Maybe TM it during a highpop round(80)

**Comment by lbnesquik** [comment]
@TiviPlus I have no clue how to test this, but it was apparently working and git was the only issue.

**Comment by BraveMole** [comment]
> @TiviPlus I have no clue how to test this, but it was apparently working and git was the only issue.

?? lower the number of points severly, and join the game to check if the job is really opened

**Comment by SpaceLoveSs13** [comment]
> 
> 
> @TiviPlus I have no clue how to test this, but it was apparently working and git was the only issue.

Tested the PR for you in local with 1 job point need. Code works fine. 
Screenshot_5

**Review by TiviPlus — APPROVED** [review]

**Comment by Naaanii** [comment]
More than one synth isn't needed.

**Pull request merged**